<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>friendWidgetItem</class>
 <widget class="QWidget" name="friendWidgetItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lable_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我是谁</string>
   </property>
  </widget>
  <widget class="QLabel" name="lable_feeling">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>分享生活留住感动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
